.dockerignore

```
@@ -8,3 +8,4 @@ venv
 .coverage
 *.swp
 *.log
+.git
```

```suggestion
**/.git
```

Already fixed in the other PR